website/docs/diff.md

```diff
@@ -6,7 +6,7 @@ slug: /diff/ # referenced in README.md and beacon
 # Known differences
 
 <!--
-   Each numbered point should have a corresponding test file in https://github.com/FerretDB/dance/tree/main/tests/diff
+   Each numbered point should have a corresponding test file in https://github.com/FerretDB/FerretDB/tree/main/integration
```

Comment:
That makes it harder to match numbered list points with unnumbered files. In a single file, tests have a relative order that could match the numbers.

Reply:
Do we want to move all of those into the single file?

Reply:
Yes